examples/framework/preferences/minimalEditor/messages/heatedSeats.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
This seems unfortunate. Perhaps we should file another issue so that we don't have to fetch empty message bundles.

Reply:
Well, the prefs editor shouldn't really be operable without message bundles - the future one won't be. The unfortunate thing is that the example was written without one, and I don't have time to update it : P

Reply:
Okay, can you file that issue instead, and we can get to it in a future release?

Reply:
https://issues.fluidproject.org/browse/FLUID-6717